### ISSUE TITLE
Plugin breaks if path contains spaces

### ISSUE THREAD

**Issue by fdw** [siouan/frontend-gradle-plugin#153]
> Before creating an issue, please:
> - Check you are using the latest plugin release.
> - Check the issue was not reported yet.

---

Hi,

I love the plugin so far, but it seems to fail if the path contains a space.

What I am trying to do: run `gradle installFrontend` in a project at `/path with spaces`
Expected results: it works
Actual results:
```
> Task installFrontend FAILED
/bin/sh: Zeile 1: /path: Datei oder Verzeichnis nicht gefunden

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task 'installFrontend'.
> Process 'command '/bin/sh'' finished with non-zero exit value 127
```

**Environment**
 
- OS: Manjaro Linux
- JDK: OpenJDK 11
- Gradle: Gradle 6.7
- Frontend Gradle plugin: 5.0.0 JDK 11

Settings in `build.gradle[.kts]` file:
```groovy
frontend {
	nodeVersion = "14.2.0"
	installScript = 'install'
	cleanScript = 'run clean'
	assembleScript = "run build"
}
```

**Comment by tngwoerleij** [comment]
At first look, this might be caused by

https://github.com/siouan/frontend-gradle-plugin/blob/cac63ce868514e2bfe1c55c6702d4a4a7f88b0b2/src/main/java/org/siouan/frontendgradleplugin/domain/usecase/ResolveExecutionSettings.java#L126

where no escaping happens compared to the Windows implementation

https://github.com/siouan/frontend-gradle-plugin/blob/cac63ce868514e2bfe1c55c6702d4a4a7f88b0b2/src/main/java/org/siouan/frontendgradleplugin/domain/usecase/ResolveExecutionSettings.java#L120-L121

I have to investigate further though.

**Comment by v1nc3n4** [comment]
Hi @fdw and @tngwoerleij,

Thank you for reporting and investigating on this issue.
I should have time this week to handle it, and find a fix. I'll keep you informed before end of next week in any case.

BR

**Comment by fdw** [comment]
Thank you! Do you already have a plan when this will be released?

**Comment by v1nc3n4** [comment]
Hello,
Release `5.0.1` is planned for mid March. The [milestones](https://github.com/siouan/frontend-gradle-plugin/milestones) section is where end-users may find information about the "release plan".
Feel free to confirm if the issue was fixed or not on your side after upgrade of course 😊.
BR

**Comment by fdw** [comment]
Cool, thank you :)

**Comment by v1nc3n4** [comment]
Hello @fdw and @tngwoerleij,

Can you confirm the issue is fixed on your side, with release 5.0.1?

Thanks in advance for your help.
BR

**Comment by tngwoerleij** [comment]
Hi @v1nc3n4, yes, I can confirm the issue is solved 👍 Thanks again!
